test/expect-tests/fsevents/fsevents_tests.ml

```diff
@@ -171,6 +171,11 @@ let test_with_multiple_fsevents ~setup ~test:f =
               Fsevents.create ~paths:[ config.dir ] ~latency:0.0
                 ~f:(make_callback sync ~f:config.on_event)
             in
+            (match config.exclusion_paths with
+            | [] -> ()
+            | paths ->
+              (* apple doesn't like [paths] empty *)
+              Fsevents.set_exclusion_paths res ~paths);
```

Comment:
It is orthogonal to this PR, but this seems like an easy trap to fall into. I would handle it specially in the C stub by not doing anything when passing an empty list.

Reply:
The problem is that it seems like a reasonable use case to unset `exclusion_paths` after setting them :/

It seems crazy that this doesn't work but I cannot find any documentation about this function online.

Reply:
Ah, indeed.